ts/torch_handler/image_segmenter.py

```diff
@@ -29,25 +33,41 @@ def preprocess(self, data):
         image = data[0].get("data")
         if image is None:
             image = data[0].get("body")
-        image = Image.open(io.BytesIO(image))
+        self.image = Image.open(io.BytesIO(image))
         trf = T.Compose([T.Resize(256),
                          T.CenterCrop(224),
                          T.ToTensor(),
                          T.Normalize(mean=[0.485, 0.456, 0.406],
                                      std=[0.229, 0.224, 0.225])])
-        image = trf(image).unsqueeze(0)
-        return image
+        return  trf(self.image).unsqueeze(0)
 
     def inference(self, data):
-        # Predict the pixel classes for segmentation
-        data = Variable(data).to(self.device)
-        pred = self.model(data)['out']
-        pred = pred.squeeze().detach().cpu().numpy()
-        return [str(pred)]
+        if torch.cuda.is_available():
+            data = data.to('cuda')
+            self.model.to('cuda')
+
+        with torch.no_grad():
+            output = self.model(data)['out'][0]
+
+        output_predictions = output.argmax(0)
+        return output_predictions
```

Comment:
I'm not sure if we should compute the argmax in here or in the `postprocess`. I would follow what is done by the `classification` examples